src/GafferUI/BackdropNodeGadget.cpp

```diff
@@ -276,8 +323,13 @@ void BackdropNodeGadget::renderLayer( Layer layer, const Style *style, RenderRea
 	bound.max /= scale;
```

Comment:
Looks like we run into trouble with scaled backdrops as we're comparing the areas of the modified bounds rather than the actual bounds?

https://github.com/GafferHQ/gaffer/assets/50844517/4e135b18-d1f6-4655-9e25-198a0b85da0b

Reply:
Good catch - fixed in https://github.com/GafferHQ/gaffer/pull/5596/commits/1fd54bde7c4528dc8ec242bcfdde6031d559baa9. I'd made another numpty error by also applying the scale in Z, which s what made the backdrop disappear in your video. Fixed that in the same commit.